tests/gh-actions/install_multi_tenancy.sh

```diff
@@ -7,3 +7,9 @@ kubectl -n kubeflow wait --for=condition=Ready pods -l kustomize.component=profi
 
 echo "Installing Multitenancy Kubeflow Roles"
 kustomize build common/kubeflow-roles/base | kubectl apply -f -
+
+echo "Installing Multitenancy Network policies"
+kustomize build common/networkpolicies/base | kubectl apply -f -
+
+echo "Enabling pod security standards for kubeflow namespace"
+kubectl label namespace kubeflow pod-security.kubernetes.io/enforce=restricted
```

Comment:
I think we should apply all the patches for all namespaces from what you added to example/kustomization.yaml

So what you referenced here. 

```
# Pod Security Standards
# https://kubernetes.io/docs/concepts/security/pod-security-standards/
# Uncomment to enable baseline level standards
# - ../contrib/security/PSS/static/baseline
# Uncomment to enable restricted level standards
# - ../contrib/security/PSS/static/restricted
# Uncomment to enable baseline level standards for dynamic namespaces
# - ../contrib/security/PSS/dynamic/baseline
# Uncomment to enable restricted level standards for dynamic namespaces
# - ../contrib/security/PSS/dynamic/restricted
```

Since they are components you need to link to all the patches referenced in the components

Reply:
Here is the reference https://kubernetes.io/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch/

Reply:
Since the rest of the patches in PSS/static/baseline are of istio-system, auth and oauth2-proxy, let's only apply the patch for the kubeflow namespace in the ./tests/gh-actions/install_multi_tenancy.sh

Reply:
Yes, then please add them to the other workflows. E.g. Istio to the istio Workflow.